app/javascript/packages/document-capture/components/in-person-prepare-step.spec.tsx

```diff
@@ -18,6 +18,22 @@ describe('InPersonPrepareStep', () => {
     ).not.to.exist();
   });
 
+  it('renders all steps to verify your identity at a Post Office', () => {
+    const { getByText } = render(<InPersonPrepareStep {...DEFAULT_PROPS} />);
+
+    expect(getByText('in_person_proofing.body.prepare.verify_step_post_office')).to.exist();
+    expect(getByText('in_person_proofing.body.prepare.verify_step_enter_pii')).to.exist();
+    expect(getByText('in_person_proofing.body.prepare.verify_step_enter_phone')).to.exist();
+    expect(getByText('in_person_proofing.body.prepare.verify_step_visit_post_office')).to.exist();
+  });
+
+  it('renders about and additional information', () => {
```

Comment:
small nit - can we update the language to `renders about step and...`

Reply:
Yes! Resolved with commit 6156de7370fe3ef82c27ce3ef53c4beb3b3e99e8